### ISSUE TITLE
Request failed with status code 500

### ISSUE THREAD

**Issue by sporbox** [jacobalberty/unifi-docker#659]
### Operating system

Ubuntu Server 22.04.2

### UniFi Tag

jacobalberty/unifi:latest

### Docker run

docker run -d --init \    --restart=unless-stopped \    -p 8080:8080 -p 8443:8443 -p 3478:3478/udp \    -e TZ='Africa/Johannesburg' \    -v ~/unifi:/unifi \    --user unifi \    --name unifi \    jacobalberty/unifi

### Bug description

Everything was going fine yesterday. When I checked in the morning, I found unifi turned itself off. When I try to start it again I get an error like this.

Request failed with status code 500

![image](https://github.com/jacobalberty/unifi-docker/assets/18421779/87067d11-a843-4d0c-8121-cc565d707308)

### Steps to reproduce

I get this error when I try to start container.

### Relevant log output

```shell
No log line matching the '' filter
```

**Comment by sporbox** [comment]
ubuntu@ubuntu:~$ sudo docker container run unifi
[sudo] password for ubuntu:
Unable to find image 'unifi:latest' locally
docker: Error response from daemon: pull access denied for unifi, repository does not exist or may require 'docker login': denied: requested access to the resource is denied.
See 'docker run --help'.

**Comment by sporbox** [comment]
docker: Error response from daemon: exec: "docker-init": executable file not found in $PATH.

**Comment by MarijanMarijanMarijan** [comment]
i have the same error since last night, 9 hours and counting. Hope all is well.

**Comment by sporbox** [comment]
I guess the source of the problem is marking the docker installation during the ubuntu installation.

**Comment by McMacaron** [comment]
> docker: Error response from daemon: exec: "docker-init": executable file not found in $PATH.
Yes, that is the error which I have. I was just about to deploy it by the customer tomorrow. Everything ready. Lucky that I check it one more time.

**Comment by McMacaron** [comment]
Guys just delete parameter --init. I guess that it has something to do with the new command "init" in docker.
In docker-compose it is "init:true".

Can somebody verify this? I am not sure why that parameter was there.

**Comment by McMacaron** [comment]
now the container is running but the status is unhealthy and I can not access web UI

**Comment by McMacaron** [comment]
Ok, I have spent the whole half of the night with it and it looks that init has incomplete binaries. I remember that I have install it by following portainer guide.  I have uninstalled docker and installed it again according to official guide.
https://docs.docker.com/engine/install/ubuntu/#install-using-the-repository
I had to delete "init: true" in docker-compose. The result is a healthy container and I can access Web UI. Still, I do not know what is the purpose to have --init in the docker file and what issue could raise from it.

**Comment by McMacaron** [comment]
One more time.
Looks that problem was cause that I had in the system two dockers one installed with "apt get" and one with "snap" package.

To summarize this:

List of docker components installed in a classic manner.
1.  $ apt list --installed | grep 'docker' 

list of all programs installed by Snap installer on your system.
2.  $ snap list 

If Docker is on both lists you have Docker in your system doubled. You have to remove one of them

$ sudo snap remove docker 

or

$ sudo apt remove docker-buildx-plugin docker-ce-cli docker-ce-rootless-extras docker-ce docker-compose-plugin
Run commands from points 1 and 2 again to check if you will find something else with the name Docker in it and remove it.

$ sudo reboot
Start from the scratch

**Comment by McMacaron** [comment]
My conclusion is that the error is caused by Snap package of Docker. Installing docker "normally" works.
https://docs.docker.com/engine/install/ubuntu/#install-using-the-repository

**Comment by sporbox** [comment]
> Bir kez daha. Sorunun snap docker paketinden kaynaklandığı görülüyor. "apt get" ile yüklenen paketler artık --init parametresi {docker-compose "init: true"} ile bile iyi çalışıyor
> 
> Bunu özetlemek gerekirse:
> 
> Docker adına göre filtrelenmiş, klasik bir şekilde yüklenen docker bileşenlerinin listesi.
> 
> 1. $ apt listesi --kurulu | grep 'liman işçisi'
> 
> Snap yükleyici tarafından sisteminize yüklenen tüm programların listesi.
> 
> 2. $ snap listesi
> 
> Docker her iki listede de yer alıyorsa sisteminizde Docker iki katına çıkar. Snap docker paketini kaldırmanız gerekiyor
> 
> * $ sudo snap docker'ı kaldır
> 
> veya ikisini birden kaldırmaya karar verirseniz
> 
> * $ sudo apt docker-buildx-plugin'i kaldır docker-ce-cli docker-ce-rootless-extras docker-ce docker-compose-plugin
> 
> İçinde Docker adında başka bir şey bulup bulamayacağınızı kontrol etmek için 1. ve 2. noktadaki komutları tekrar çalıştırın ve onu kaldırın.
> 
> * $ sudo yeniden başlatma
> 
> Sıfırdan başla

**Comment by miknuc** [comment]
Is there any other solution than replacing the snap docker version? I have several containerized solutions running on my machine and I can't exchange the container runtime because a single stack has an issue. 
BTW: I am only using the snap version of docker, 

`apt list --installed | grep docker`
 is empty.
